### ISSUE TITLE
PHP open/close tags inside CSS files are replaced with internal PHPCS token strings when auto fixing

### ISSUE THREAD

**Issue by dakanji** [squizlabs/PHP_CodeSniffer#2467]
PHP tokens are changed to program tokens but are not changed back when the appear in comment blocks

#### PHP version:
PHP 7.1.1 (cli) (built: Feb 13 2017 10:05:49) ( NTS )

#### PHP_CodeSniffer version:
PHP_CodeSniffer version 3.4.0 (stable) by Squiz (http://www.squiz.net)

#### Minimum samples of PHP code:
 * ... before running PHP_CodeSniffer:
```php
/* <?php $varName = $ourClass->ourFunc('Some Test'); ?>
 * Some Txt
 */
```

 * ... with unexpected changes applied when running PHP_CodeSniffer:
```php
/* ^PHPCS_CSS_T_OPEN_TAG^ $varName = $ourClass->ourFunc('Some Test'); ^PHPCS_CSS_T_CLOSE_TAG^
 * Some Txt
 */
```

 * .... with the changes expected:
```php
// No Change
```

**Comment by dakanji** [comment]
PS. Only seems to happen when the commented PHP code is in a `*.css` file

**Comment by gsherwood** [comment]
This will be fixed in 3.4.2. Thanks for reporting it.
